helix-term/src/ui/info.rs

```diff
@@ -21,7 +21,7 @@ impl Component for Info {
             width,
             height,
         ));
-        surface.clear_with(area, popup_style);
+        surface.clear_with(area, text_style);
```

Comment:
It looks like this is an unrelated change? I believe `popup_style` is the correct style to clear with here anyways

Reply:
oh, yes, this is an unrelated change, it's my mistake. I will revert it.